siriuspy/siriuspy/idff/main.py

```diff
@@ -45,6 +46,7 @@ def __init__(self, idname):
             'LoopState-Sel': self.set_loop_state,
             'LoopFreq-SP': self.set_loop_freq,
             'ControlQS-Sel': self.set_control_qs,
+            'ControlQN-Sel': self.set_control_qn,
```

Comment:
could we name this as ControlQTrim/control_qtrim?

Reply:
ControlQS-Sel and ControlQTrim-Sel ? QN is a shortcut for normal quadrupole variation, be it with a trim ps or other method. I prefer the more symmetrical ControlQS-Sel and ControlQN-Sel.

Reply:
I see, but we generally don't use this in other parts of the control system... And another point is that we will never control a main quadrupole power supply in a specific ID feedforward, only local trims

Reply:
I also prefer the QN option here. Maybe we never used this nomenclature in the control system because we didn't need to separate the normal from the skew quads, or maybe it was more important to separate trim quads from family quads... But in this case, since it is not necessary to separate trim from fam, the name QN is better suited. 

Another argument to support this is that skew quads are also trim coils in the sexts. In this sense, the name QTrim would not separate skews from normal quads for those not familiar with the control system.